functions.php

```diff
@@ -1319,3 +1340,287 @@ function wp_autoupdates_themes_bulk_actions_handle( $redirect_to, $doaction, $it
 	}
 }
 add_action( 'handle_network_bulk_actions-themes-network', 'wp_autoupdates_themes_bulk_actions_handle', 10, 3 );
+
+/**
+ * Disable auto updates via Ajax.
+ */
+function wp_autoupdates_disable_auto_updates() {
+	$nonce = sanitize_text_field( $_POST['nonce'] );
```

Comment:
I’m a bit worried about using `sanitize_text_field` as it's only sanitization. Shouldn't we also escape unwanted HTML bits?

Pinging @whyisjake as WordPress security team maintainer for further consideration.

Reply:
Yeah, great idea.

Reply:
Coding standards want me to use `wp_unslash` and then a sanitization before even touching the variable. But I'll let the security team come up with the suggestions.